.husky/pre-commit

```diff
@@ -18,5 +18,8 @@ npm run lint:fix
 
 npm run gen:schema
 
+# Generate table of contents for all markdown files
+npm run generate:toc
```

Comment:
This should specify only the markdown in the root directory. How can that be done?